src/MoBi.Core/Commands/UpdateSimulationCommand.cs

```diff
@@ -46,6 +51,11 @@ public UpdateSimulationCommand(IMoBiSimulation simulationToUpdate, IModel newMod
          _changedBuildingBlockType = changedBuildingBlockType;
          ObjectType = ObjectTypes.Simulation;
          CommandType = AppConstants.Commands.UpdateCommand;
+
+         if (string.IsNullOrEmpty(_changedBuildingBlockName))
```

Comment:
We had all the logic already to update a single building block. I am just ensuring that we reuse the same logic for configuration by changing the description of the command